recipes-graphics/userland/userlandtools_git.bb

```diff
@@ -0,0 +1,33 @@
+require userland.inc
+
+RCONFLICTS_${PN} = "userland"
+
+# Keep only those libs & bins that are actually used during EEPROM update
+do_install_append () {
```

Comment:
This looks a bit hacky - can't we just overwrite the install task manually pulling in only the needed blobs?